transport/ca/localca/signer.go

```diff
@@ -146,13 +148,22 @@ func New(req *csr.CertificateRequest, profiles *config.Signing) (*CA, error) {
 	// CFSSL has become inconsistent, and it can't be trusted.
 
 	priv, err := helpers.ParsePrivateKeyPEM(keyPEM)
-	assert.NoError(err, "CFSSL-generated private key can't be parsed")
+	if err != nil {
+		println("CFSSL-generated private key can't be parsed")
```

Comment:
since the function signature supports it, I think returning an error would be more idiomatic than panic (or os.exit for that matter) 
```suggestion
		return nil, fmt.Errorf("CFSSL-generated private key can't be parsed: %w", err)
```

Reply:
Yup, works for me! I tried to stay close to the original approach, but agreed that if we return nil, nobody would be able to use the result anyway. Let me update.